moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -37,6 +37,8 @@
 #include <ros/ros.h>
 #include <chomp_motion_planner/chomp_trajectory.h>
 #include <iostream>
+#include <stdio.h>
+using namespace std;
```

Comment:
as a convention we do use ``using namespace std``

Reply:
thanks for the info.

Reply:
to clarify: can you remove it from usage here?

Reply:
sure, removed